l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -120,8 +136,14 @@ def open(self, mode: str = "r", cached: bool = True, cache_size_bytes: int = int
         self.frames = self.root[FRAME_ARRAY_KEY]
         self.agents = self.root[AGENT_ARRAY_KEY]
         self.scenes = self.root[SCENE_ARRAY_KEY]
+        try:
+            self.tl_faces = self.root[TL_FACE_ARRAY_KEY]
+        except KeyError:
+            print(f"{TL_FACE_ARRAY_KEY} not found in {self.path}! Traffic lights will be disabled")
+            self.tl_faces = np.empty((0,), dtype=TL_FACE_DTYPE)
 
     def __str__(self) -> str:
+        # TODO add traffic faces
```

Comment:
Should we add that?

Reply:
I'd say not now because it's not clear to me what we want to print about it atm